src/WebJobs.Script.Grpc/Abstractions/WorkerDescriptionExtensions.cs

```diff
@@ -21,10 +21,6 @@ public static void Validate(this WorkerDescription workerDescription)
             {
                 throw new ValidationException($"WorkerDescription {nameof(workerDescription.DefaultExecutablePath)} cannot be empty");
             }
```

Comment:
Add test in [GenericWorkerProviderTests.cs](https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Rpc/GenericWorkerProviderTests.cs) that verifies empty/null defaultWorkerPath. Here is the code that builds a test config [GetTestDefaultWorkerDescription](https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Rpc/GenericWorkerProviderTests.cs#L374)